packages/runtimes/hmr/src/loaders/hmr-runtime.js

```diff
@@ -139,12 +140,19 @@ if ((!parent || !parent.isParcelRequire) && typeof WebSocket !== 'undefined') {
 
     var data /*: HMRMessage */ = JSON.parse(event.data);
 
-    if (data.type === 'update') {
+    if (data.type === 'partial-update') {
```

Comment:
Is there a potential edge case where we "lose" the update message and are left with incomplete / incorrect data here? Should a groups of `partial-update` messages have some sort of unique identifier? We know how many partial-updates we're going to send so they could even contain an `x of y` type thing.

Not sure what the delivery reliability for websockets is like, and whether it's worth the additional complexity.

Reply:
I initially didn't want to add that complexity because I assumed that since WebSockets are TCP-based, we can expect correct order and delivery. That may be a poor assumption, and I am happy to refactor if required.

Reply:
Yeah, a quick look shows that websocket messages should arrive in the correct order (and won't be "lost" unless the actual connection dies).. I think it's fine as is, and if we have actual issues we can work out a suitable solution then.